### PR TITLE
League stats jake

### PR DESCRIPTION
**Pull request by jakeheft**
This PR will:
- Add method count_of_teams
- Add method (and helpers) for highest_scoring_visitor
- Add method (and helpers) for highest_scoring_home_team
- All tests passing

### CODE REVIEW

**Review by dunlapww — APPROVED** [review]
Hi Jake, I see you call avg_score method, but didn't think i'd uploaded that to main yet.  but

**Comment by jakeheft** [comment]
Will, I stole avg_score method from you 😉

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -142,6 +142,51 @@ def worst_team(season)
...
+  def avg_score(filtered_game_teams = @game_teams)
+    ratio(total_score(filtered_game_teams), total_game_teams(filtered_game_teams))
+  end
+
+  def team_id_to_team_name(id)
+    team_obj = @teams.select do |team|
+      team.team_id == id
+    end
+    team_obj[0].team_name
+  end
```
Comment (hopesgit) [inline lib/stat_tracker.rb:159]: I did already make this method, so it's up to you if you wanna keep it. (for reference: it's "team_names_by_team_id")

Comment (jdcarey128) [reply lib/stat_tracker.rb:159]: hahah same here, let's keep all them and then refactor later

**Review by jdcarey128 — APPROVED** [review]

**Pull request merged**